tests/budget_accounting_test.py

```diff
@@ -330,58 +330,58 @@ class ComputeBudgetTestCase:
                 ],
                 expected_pipeline_noise_std=0.9428090415820634),
             ComputeBudgetTestCase(name="laplace_mechanisms",
-                                  epsilon=0.19348133991361996,
+                                  epsilon=0.168,
                                   delta=1e-3,
                                   mechanisms=[
                                       ComputeBudgetMechanisms(
                                           count=10,
-                                          expected_noise_std=50,
+                                          expected_noise_std=49.872,
```

Comment:
Did the computations become more precise?

Reply:
Yes